python/lsst/analysis/ap/utilities.py

```diff
@@ -0,0 +1,70 @@
+# This file is part of analysis_ap.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["makeSimbadLink"]
+
+from IPython.display import display, Markdown
+import astropy.coordinates as coord
+from astroquery.simbad import Simbad
+import astropy.units as u
+
+
+def makeSimbadLink(source, arcsec_radius=3.0, make_table=False):
+    """Search Simbad for associated sources within a 3 arcsecond region
+        Maximum search radius should not be greater than 5 arcseconds.
+
+    Parameters
+    ----------
+    source : 'pandas.core.series.Series'
+        Source pandas dataframe that include the ra and dec of the source.
+
+    arcsec_radius : 'float'
+        Search radius submitted to Simbad in arcseconds.
+        Default radius is 3 arcseconds.
+
+        Returns
+    -------
+    results_table : `astropy.table.table.Table`
+        A table of Simbad search results from astro_query.
+    """
+    ra = source["ra"]
+    dec = source["decl"]
+    search_results = f"http://simbad.cds.unistra.fr/simbad/sim-coo?Coord={ra}+{dec}" \
+                     f"&CooFrame=FK5&CooEpoch=2000&CooEqui=2000&CooDefinedFrames=none&Radius=" \
+                     f"{arcsec_radius}&Radius.unit=arcsec&submit=submit+query&CoordList="
+    display(Markdown(f"[Link to Simbad search]({search_results})"))
```

Comment:
What happens if you run this function just in a command-line Python interpreter?  If it's only for use in the notebook that's totally reasonable but we should say so in the docstring.

Reply:
Hmm... If we are making functions only for use in notebooks, we should maybe put them somewhere other than `utils.py`, which is generally assumed to be generic. `nb_utils.py`, though I don't really like that.

Reply:
On the idea of notebook or not notebook, I could make this be both command line friendly and notebook friendly with a key setting (nb=False). Or, should a separate command line only utility be made? In the command line, what would the desired output be? A text file containing all of the output tables?

Reply:
I don't have a strong opinion as to the right approach here.  I'm curious what happens to the `display` output in a script or the IPython CLI though.

Reply:
I tried it out and it returns <IPython.core.display.Markdown object> from the python command line, so it doesn't break it but definitely doesn't create a link. The table is still returned.